Solutions/Authomize/Analytic Rules/Privileged_Machines_Exposed_to_the_Internet.yaml

```diff
@@ -0,0 +1,59 @@
+id: 72891de4-da70-44e4-9984-35fcea98d000
+name: Privileged Machines Exposed to the Internet
+kind: Scheduled
+description: These are AWS Ec2 machines that are exposed to the internet. You can further filter by tags so that you can, for example, find exposed machines that are also "privileged".
+severity: High
+queryFrequency: 5m
+queryPeriod: 30m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+- Discovery
+- Impact
+relevantTechniques:
+- T1580
+query: |-
+  union withsource=_TableName Authomize_v2_CL
```

Comment:
@sriley0975 any specific reason of using union with just one table?

Reply:
No - system auto generated and I didn't change - I will update this in all the files and clean up KQL.